src/App.tsx

```diff
@@ -27,35 +34,101 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const LanguageSelector = (props: LanguageSelectorProps): ReactElement => {
-  const [name, setName] = useState(props.name);
-  const classes = useStyles();
-  const tStyle = { color: 'white' };
+const enumStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 100,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const LanguageSelector = (): ReactElement => {
+  const [name, setName] = useState(supportedLanguages[langStore.getItem('lang') as string].name);
+  const classes = langStyles();
+  const textStyle = { color: "white" };
```

Comment:
this is good, but the best way to do this is to add a class name to the object passed to the makeStyle function. The same way we do for the class names 'formControl' and 'selectEmpty'.

Reply:
The reason to use className is that you can share the class names across multiple components. So if you want to change the background color, instead of having to change the textStyle = {color: "white"} in multiple place, you just change the value assigned to the class name in one location.